zebra/zebra_vxlan.h

```diff
@@ -26,6 +26,16 @@
 extern "C" {
 #endif
 
+DECLARE_MGROUP(ZEBRA);
+
+DECLARE_MTYPE(VLAN_CHANGE_ARR);
```

Comment:
why put the DECLARE here , but the DEFINE in another module? I think it's clearer if the mtype DECLARE and DEFINE are in related modules. And since the MGROUP is already declared in rib.h, that would probably be unnecessary there?

Reply:
I think this DECLARE is redundant here.  Will move the DEFINE accordingly to dplane.h